controller_manager/src/controller_manager.cpp

```diff
@@ -318,7 +318,8 @@ controller_interface::return_type ControllerManager::configure_controller(
     RCLCPP_ERROR(
       get_logger(),
       "after configuring, controller %s is in state %s, expected Inactive",
```

Comment:
nitpick - capitalize After

Reply:
Good catch!

There are many of those "capitalization" issues. We should at least decide if we start with small or capitalized letters.